tests/pytest/test_admin.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
@thekaveman here's what i have so far:

- make an AdminUser fixture
- mocking `token` and _trying_ to mock the Google SSO response and json object - but the mock/patch isn't returning the values I want.

Basically trying to test: That before the `pre_login_user` method, a user has a blank first and last name. After the method runs, the user's first and last name come from the Google SSO response json object. Should I be taking a different approach? (Like just making sure the `pre_login_user` method gets called when a user logs in.. or something like that?)

Reply:
@thekaveman Good news - after refactoring the tests to use the `mocker` pattern, the tests are now passing as expected 👍 At least locally.

Now getting GitHub Action CI failures: https://github.com/cal-itp/benefits/actions/runs/7793576156/job/21253511521#step:8:50 

Not really sure why/how?? https://stackoverflow.com/questions/40206569/django-model-doesnt-declare-an-explicit-app-label

Reply:
Arg, I'm running into a similar issue with that same error, trying to test a custom field I'm creating for #1847.

I've been trying to find a solution for about the last hour but nothing yet :angry:

Reply:
I think at this point with all these problems running unit tests that need database/model access, we can probably just call this PR good to go. If you can just interactive rebase to drop the 3 commits related to testing [660f269](https://github.com/cal-itp/benefits/pull/1855/commits/660f26997a7e169cbbcaf6c4d2e03904b8d3a8e0), [0d5984e](https://github.com/cal-itp/benefits/pull/1855/commits/0d5984ef1b2c47321b13a4f0709989795c501723), and [2e28ec7](https://github.com/cal-itp/benefits/pull/1855/commits/2e28ec7de000169eba060b332e56519e2046547c), and then [move the `admin.py` file as you suggested](https://github.com/cal-itp/benefits/pull/1855#discussion_r1479147581), this can be approved for merge.

Reply:
The problem in that GitHub Action failure is once again because some apps are installed [only if `ADMIN` is enabled](https://github.com/cal-itp/benefits/blob/dev/benefits/settings.py#L44). 

We decided that the ultimate change we want to make regarding `ADMIN` is to remove it entirely (#1856), but it seems like if we went ahead and just removed this one `ADMIN` check, i.e. install all the apps, then the unit test @machikoyasuda wrote will work and hopefully whatever issue @thekaveman  is seeing in #1847 will also go away.

Reply:
My issue was unrelated to the `ADMIN` setting, it was in trying to unit test a custom field by attaching it to a model that doesn't exist in our migrations (just a model I defined directly in the tests).

@machikoyasuda timebox to about 30 minutes here, if you can get it working with minimal impact/changes then great, otherwise don't worry about the tests for now. Let's get this wrapped up.